tests/support/test_opack.py

```diff
@@ -36,8 +36,7 @@ def test_pack_uuid():
 
 
 def test_pack_absolute_time():
-    with pytest.raises(NotImplementedError):
-        pack(datetime.now())
+    assert pack(datetime.fromtimestamp(978307200.0)) == b"\x06" + (b"\x00" * 8)
```

Comment:
Same with the tests, I don't really like magic constants. I have used mach time in MRP IIRC, so you could add some kind of helper method to convert `datetime` objects mach time (and possibly pack) if you like.